spring-cloud-starter-aws-parameter-store-config/src/test/java/org/springframework/cloud/aws/autoconfigure/paramstore/AwsParamStoreBootstrapConfigurationTest.java

```diff
@@ -35,42 +39,53 @@
  */
 class AwsParamStoreBootstrapConfigurationTest {
 
-	AwsParamStoreBootstrapConfiguration bootstrapConfig = new AwsParamStoreBootstrapConfiguration();
+	private final ApplicationContextRunner contextRunner = new ApplicationContextRunner()
+			.withConfiguration(
+					AutoConfigurations.of(AwsParamStoreBootstrapConfiguration.class));
 
 	@Test
 	void testWithStaticRegion() {
-		String region = "us-east-2";
-		AWSSimpleSystemsManagementClient awsSimpleClient = createParamStoreClient(region);
+		this.contextRunner.withPropertyValues("aws.paramstore.enabled:true",
+				"aws.paramstore.region:us-east-2").run(context -> {
+					assertThat(context.getBean(AwsParamStorePropertySourceLocator.class))
```

Comment:
This and all the other occurrences could be improved by using dedicated method to check if bean is there:
```
assertThat(context).hasSingleBean(AwsParamStorePropertySourceLocator.class);
```

Reply:
Something like this now?

Reply:
Yep, exactly!